src/main/java/com/appland/appmap/record/Recorder.java

```diff
@@ -138,6 +135,20 @@ private void queueEvent(Event event) throws ActiveSessionException {
 
       recordingSession = this.activeSession;
       pendingEvent = this.queuedEvents.get(event.threadId);
```

Comment:
I didn't changed the current imploementation, of pending and queued events, but now it seems that is not needed, what do you think @dustinbyrne ?

Reply:
@dustinbyrne could you check this